events/world-moveit-day-2018/index.markdown

```diff
@@ -0,0 +1,102 @@
+- [PickNik Consulting](http://picknik.ai), Boulder, USA
+  - Contact: Dave Coleman
+- [Fraunhofer IPA](https://www.ipa.fraunhofer.de/en.html) / [RIC-EU](https://rosindustrial.org/ric-eu/), Stuttgart, Germany
+  - Contact: Michael 'v4hn' Görner
+- [ROS-Industrial Asian Pacific Consortium](http://rosindustrial.org/ric-apac/), Singapore
+  - October 26
+  - Contact: Chan Min Ling
+- [Tokyo Opensource Robotics Kyokai Association (TORK)](http://opensource-robotics.tokyo.jp/), Tokyo, Japan
+  - October 26
+  - Contact: Ryosuke Tajima
+- [Southwest Research Institute](https://www.swri.org/), San Antonio, USA
+  - Contact: Matt Robinson
+
+## Organizers
+
+  * Dave Coleman, Nathan Brooks, Rob Coleman // PickNik Consulting
+
+## Signup
+
+Please state your intent to join the event [on this form](https://docs.google.com/forms/d/e/1FAIpQLSdk_xMOVdqusdvT6vr5s6AnXV-GdtchCjrX-BXpyw642ahcBg/viewform). Note that specific locations will have their own signups in addition to this form.
+
+If you aren't near an organized event we encourage you to have your own event in your lab/organization/company and video conference in to all the other events. We would also like to mail your team or event some MoveIt! stickers to schwag out your robots!
+
+## Logistics
+
+### What version of MoveIt! should you use?
+
+We recommend the Kinetic LTS branch/release. The Melodic release is also a good choice but is new and has been tested less. The Indigo branch is considered stable and frozen - and only critical bug fixes will be backported.
+
+For your convenience, a VirtualBox image for ROS Kinetic on Ubuntu 16.04 is available [here](http://backend.picknik.ai/).
+
+### Finding Where You Can Help
+
+Suggested areas for improvement are tracked on the new unified moveit repo via several labels:
+
+- [moveit day candidate](https://github.com/ros-planning/moveit/labels/moveit%20day%20candidate) labels issues as possible entry points for participants in the event. This list will grow longer before the event.
+- [simple improvements](https://github.com/ros-planning/moveit/issues?q=is%3Aopen+-label%3Aassigned+label%3A%22simple+improvements%22) indicates the issue can probably be tackled in a few hours.
+- [documentation](https://github.com/ros-planning/moveit/issues?utf8=%E2%9C%93&q=is%3Aopen%20label%3Adocumentation%20-label%3Aassigned%20) suggests new tutorials, changes to the website, etc.
+- [assigned](https://github.com/ros-planning/moveit/issues?q=is%3Aopen+is%3Aissue+label%3Aassigned) aids developers to find issues that are not already being worked on.
+- **no label** - of course issues that are not marked can still be worked on during World MoveIt! day, though they will likely take longer than one day to complete.
+
+If you would like to help the MoveIt! project by tackling an issue, claim the issue by commenting "I'll work on this" and a maintainer will then hopefully add the label "assigned". Feel free to ask further questions in each issue's comments.
```

Comment:
remove "then hopefully".

Maybe also add 'The developers will aim to reply to WMD-related questions' in time before the event.